Assets/Huawei/Editor/View/PublishingAPI/QueryingAppInfoEditor.cs

```diff
@@ -0,0 +1,315 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+using UnityEditor;
+using System;
+using UnityEngine.Networking;
+using System.Linq;
+
+namespace HmsPlugin.PublishingAPI
+{
+    public class QueryingAppInfoEditor : VerticalSequenceDrawer
+    {
+        private Label.Label ReleaseState;
+        private Label.Label AppName;
+        private int ParentType;
+        private int ChildType;
+        private int GrandChildType;
+        private CategoryLevel categoryLevel;
+        private Label.Label parentTypeString;
+        private Label.Label childTypeString;
+        private Label.Label grandChildTypeString;
+
+
+        //TODO: AccessToken ve ClientID aldıktan sonra App Information için query yap ve gelenleri ekrana set et.
+        // GetAccessTokenAsync func ile client id ve access token al
+
+        public QueryingAppInfoEditor()
+        {
+
+            categoryLevel = new CategoryLevel();
+
+            categoryLevel.Level1 = new List<KeyValuePair<int, string>>();
+            categoryLevel.Level1.Add(new KeyValuePair<int, string>(-1, "Not Assigned"));

```

Comment:
Refactor category levels to be in a method then call that method in constructor.